images/spin-inbound-redis/spin.toml

```diff
@@ -10,7 +10,7 @@ redis_address = { required = true }
 redis_channel = { required = true }
 
 [[trigger.redis]]
-address = "redis://redis-service.default.svc.cluster.local:6379"
+address = "{{ redis_address }}"
```

Comment:
Question: Where is the value of `redis_address` passed to Spin?

Reply:
The spin operator configures them by setting them in an environment variable prefixed with `SPIN_VARIABLE_` [here](https://github.com/spinkube/spin-operator/blob/0b88d612bbd0e3b1617db4569a0c1e0fb900be35/internal/controller/deployment.go#L86). Spin looks for env vars with that prefix and exposes them through the variables API. 

They can be set in the `spec.variables` section of a [SpinApp CRD](https://www.spinkube.dev/docs/spin-operator/reference/spin-app/#spinappspecvariablesindex).

Reply:
Ah right. thanks!